app/controllers/providers_controller.rb

```diff
@@ -14,11 +14,13 @@ def index
     )
 
     render jsonapi: @providers.vendors.to_a,
-           meta: { totalResults: @providers.totalResults }
+           meta: { totalResults: @providers.totalResults },
+           class: { Provider: SerializableProviderList }
```

Comment:
It might do with a brief comment here to show that we're using a different serializer here to work around a bug in RMAPI, and that we can use the normal serializer once that fix has been deployed. It would be a major bonus to have some way to reference the status of the bug so that someone coming along later.